contracts/ReputationMiningCycle.sol

```diff
@@ -555,7 +577,11 @@ contract ReputationMiningCycle is PatriciaTreeProofs, DSMath {
     }
   }
 
-  function getExpectedSkillIdAndAddress( ReputationLogEntry storage logEntry, uint256 updateNumber ) internal returns (uint256 expectedSkillId, address expectedAddress) {
+  function getExpectedSkillIdAndAddress(ReputationLogEntry storage logEntry, uint256 updateNumber)
+  internal
+  view
```

Comment:
Put `internal view` at the end of the function declaration line above. That's what we've followed everywhere else I think. The `returns..` can be on its own line to stand out